test/e2e/baseline_test.go

```diff
@@ -691,3 +697,142 @@ func SetWaypoint(t framework.TestContext, ns string, name string, waypoint strin
 		}
 	}
 }
+
+func TestL4Telemetry(t *testing.T) {
+	framework.NewTest(t).Run(func(tc framework.TestContext) {
+		for _, src := range apps.All {
+			for _, dst := range apps.All {
```

Comment:
```suggestion
                  for _, src := range apps.EnrolledToKmesh {
			for _, dst := range apps.EnrolledToKmesh {
```

`apps.All` contains all test applications. Although they are all currently managed by Kmesh, we may include unmanaged applications in the future. So use `apps.EnrolledToKmesh`  is sufficient, which means the applications managed by Kmesh